src/Components/Politicians/PoliticianCard/index.js

```diff
@@ -1,23 +1,18 @@
 /* @flow */
 import React from 'react';
-import { Card, CardHeader, CardText } from 'material-ui/Card';
-import LinearProgress from 'material-ui/LinearProgress';
+import { Card, CardHeader } from 'material-ui/Card';
 
 export type Props = {
   name: string,
   party: string,
-  progress: number,
 };
 
 const PoliticianCard = (props: Props) => {
-  const { name, party, progress } = props;
-  const cardStyle = {};
+  const { name } = props;
+  const cardStyle = { flex: '1 1 0', margin: '1%' };
```

Comment:
Nitpick:

I guess you could define this (and any static styles) outside of the component.
Either in a separate file or just outside of the `PoliticianCard` function